src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -2173,6 +2174,7 @@ LedgerMaster::makeFetchPack(
             << "Built fetch pack with " << reply.objects().size() << " nodes";
         auto msg = std::make_shared<Message>(reply, protocol::mtGET_OBJECTS);
         peer->send(msg);
+        m_stats.inboundFetchPackRequests++;
```

Comment:
The meaning of `outboundFetchPackRequests` is clear to me. But I am not sure about `inboundFetchPackRequests`. Since it is increased here, we don't count all inbound requests. We only count the requests with replies. 

I just want to confirm what is being measured, the count of inbound requests or the count of replies?

Reply:
Yes, my intention was to only count requests with replies. I will be sure to note this in the documentation of the counter. Would you recommend that I change the name of the variable? I could also make a separate counter for all inbound requests if that is required.

Reply:
Now that @pwang200 has asked the question, I'm a little confused. When you say "requests with replies", I take that to mean "inbound requests for which we were able to generate a reply". Is that correct? If so, I'd suggest renaming this counter to `outboundFetchPackReplies`.

And I do think it would be worthwhile to create a different `inboundFetchPathReplies` which counts the number of replies to our outbound queries.

Reply:
@mayurbhandary I think the variable should be renamed. And please document the counters clearly.

Reply:
Closing this PR and creating a new one to address these comments. New PR will also include the full set of fetchpack metrics for inbound and outbound requests/replies.